cl/recap/tasks.py

```diff
@@ -1571,6 +1572,16 @@ def fetch_pacer_doc_by_rd(
         mark_fq_status(fq, msg, PROCESSING_STATUS.FAILED)
         self.request.chain = None
         return
+    except PacerLoginException as exc:
```

Comment:
I'm surprised we don't need to do this in about a dozen other places?

Reply:
Sentry reports indicate the `PacerLoginException` originates from the `fetch_pacer_doc_by_rd`, `fetch_docket`, `fetch_attachment_page`, or the `get_and_process_free_pdf` task. While [`fetch_docket`](https://github.com/freelawproject/courtlistener/blob/3a4baf03aed162d6c0d22dcb4cf62072f11ff566/cl/recap/tasks.py#L1829-L1840) and [`get_and_process_free_pdf`](https://github.com/freelawproject/courtlistener/blob/3a4baf03aed162d6c0d22dcb4cf62072f11ff566/cl/corpus_importer/tasks.py#L626-L636) already handle the exception, I've added the same logic to the `fetch_attachment_page` task for consistency.

Thanks for your comment.